src/models/Token.model.js

```diff
@@ -0,0 +1,17 @@
+const { DataTypes } = require('sequelize');
+const { client } = require('../db');
+const { User } = require('./User.model');
+
+const Token = client.define('Token', {
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
Similarly, the `hasOne` association should specify the foreign key explicitly if it does not follow the default naming convention. Consider adding `{ foreignKey: 'userId' }` to the `hasOne` method to ensure the correct foreign key is used.